pipelines/rj_ssm__celular_seguro/flow.py

```
@@ -0,0 +1,30 @@
+from iplanrio.pipelines_utils.env import getenv_or_action
+from prefect import flow
+
+
+@flow(log_prints=True)
+def rj_ssm__celular_seguro():
+    url = getenv_or_action("API_SINESP__ACESS_TOKEN_URL")
+    # api_key = getenv_or_action("API_SINESP__CLIENT_ID")
+    client_id = getenv_or_action("API_SINESP__CLIENT_ID")
+    client_secret = getenv_or_action("API_SINESP__CLIENT_SECRET")
+
+    result = os.system(
```

Ruff (0.15.20)

[error] Starting a process with a shell, possible injection detected

(S605)